app/models/account.rb

```diff
@@ -37,6 +37,17 @@ def self.canonical_cname(cname)
     cname
   end
 
+  # Change the current active Account.
+  # Required until https://github.com/influitive/apartment/pull/307 is merged
+  # @param [String] cname the host name for the account
+  # @example
+  #   Account.use_account!('whatever.somewhere.io')
+  def self.use_account!(cname)
```

Comment:
I prefer the class method approach here. It says, "I don't care what instance you currently have; just use this named account." I agree about the potential for confusion with `switch!` if defined as an instance method.